operations/node_management/Rebuild_NCNs.md

```diff
@@ -130,168 +136,159 @@ Skip this section if rebuilding a master or storage node. The examples in this s
 
-   ```bash
-   ncn# cray cfs components describe XNAME --format json
-   {
-     "configurationStatus": "configured",
-     "desiredConfig": "ncn-personalization-full",
-     "enabled": true,
-     "errorCount": 0,
-     "id": "x3000c0s7b0n0",
-     "retryPolicy": 3,
-   ```
+    ```bash
+    ncn-m# cray cfs components describe $XNAME --format json
+    {
+      "configurationStatus": "configured",
+      "desiredConfig": "ncn-personalization-full",
+      "enabled": true,
+      "errorCount": 0,
+      "id": "x3000c0s7b0n0",
+      "retryPolicy": 3,
+    ```
 
-   If the configurationStatus is `pending`, wait for the job finish before rebooting this node. If the configurationStatus is `failed`, this means the failed CFS job configurationStatus preceded this worker rebuild, and that can be addressed independent of rebuilding this worker. If the configurationStatus is `unconfigured` and the NCN personalization procedure has not been done as part of an install yet, this can be ignored.
+    If the configurationStatus is `pending`, wait for the job finish before rebooting this node. If the configurationStatus is `failed`, this means the failed CFS job configurationStatus preceded this worker rebuild, and that can be addressed independent of rebuilding this worker. If the configurationStatus is `unconfigured` and the NCN personalization procedure has not been done as part of an install yet, this can be ignored.
 
 1. Determine if the master node being rebuilt is the first master node.
 
-   The first master node is the node others contact to join the Kubernetes cluster. If this is the node being rebuilt, promote another master node to the initial node before proceeding. Run the following command from any master node:
-
-   ```bash
-   ncn# craysys metadata get first-master-hostname
-   ncn-m002
-   ```
+    The first master node is the node others contact to join the Kubernetes cluster. If this is the node being rebuilt, promote another master node to the initial node before proceeding.
 
-   If the node returned is not the one being rebuilt, proceed to the step which [stops etcd](#stop-etcd) and skip the substeps here.
+    ```bash
+    ncn-m# craysys metadata get first-master-hostname
+    ncn-m002
+    ```
 
-   1. Reconfigure the Boot Script Service \(BSS\) to point to a new first master node.
+    If the node returned is not the one being rebuilt, proceed to the step which [stops etcd](#stop-etcd) and skip the substeps here.
 
-      Run this step on a master or worker node that is not being rebuilt.
+    1. Reconfigure the Boot Script Service \(BSS\) to point to a new first master node.
 
-      ```bash
-      ncn# cray bss bootparameters list --name Global --format=json | jq '.[]' > Global.json
-      ```
+        ```bash
+        ncn-mw# cray bss bootparameters list --name Global --format=json | jq '.[]' > Global.json
```

Comment:
The ncn-mw# prompt used in this procedure isn't documented as a node abbreviation: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#typographic-conventions 

It might be helpful to include a note towards the start of the procedure that indicates what this prompt means to help avoid confusion.